pkg/k8s/apis/cilium.io/v2alpha1/types.go

```diff
@@ -72,3 +74,55 @@ type EgressRule struct {
 	// selector semantics; if present but empty, it selects all pods.
 	PodSelector *slimv1.LabelSelector `json:"podSelector,omitempty"`
 }
+
+// CoreCiliumEndpoint is slim version of status of CiliumEndpoint.
+type CoreCiliumEndpoint struct {
+	// Name indicate as CiliumEndpoint name.
+	Name string `json:"name,omitempty"`
+	// IdentityID is the numeric identity of the endpoint
+	IdentityID int64 `json:"id,omitempty"`
+	// Networking is the networking properties of the endpoint.
+
+	// +kubebuilder:validation:Optional
+	Networking *cilium_v2.EndpointNetworking `json:"networking,omitempty"`
+	// Encryption is the encryption configuration of the node
+
+	// +kubebuilder:validation:Optional
+	Encryption cilium_v2.EncryptionSpec `json:"encryption,omitempty"`
+	NamedPorts models.NamedPorts        `json:"named-ports,omitempty"`
+}
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:resource:categories={cilium},singular="ciliumendpointslice",path="ciliumendpointslices",scope="Cluster",shortName={ces}
+// +kubebuilder:storageversion
+
+// CiliumEndpointSlice is the status of a group of CoreCiliumendpoints.
```

Comment:
"status" is a bit misleading (because neither CoreCEP nor CES has status), maybe "contains a set of CoreCiliumendpoints"?

Reply:
Done.